demo/CMakeLists.txt

```diff
@@ -4,15 +4,17 @@ project (PegasDemo)
 set (CMAKE_CXX_STANDARD 17)
 
 #########################################################
-# FIND GLUT
+# ADD FreeGLUT
 #########################################################
-find_package(GLUT REQUIRED)
-include_directories(${GLUT_INCLUDE_DIRS} ${GLUT_INCLUDE_DIR})
-link_directories(${GLUT_LIBRARY_DIRS})
-add_definitions(${GLUT_DEFINITIONS})
-if(NOT GLUT_FOUND)
-    message(ERROR " GLUT not found!")
-endif(NOT GLUT_FOUND)
+set(FREEGLUT_BUILD_DEMOS OFF)
```

Comment:
Personally I would create a special `GlutConfig.cmake` file and set all needed options for add_subdirectory + create new variables to use in your code so you would use it as this:

```cmake
include(GlutConfig)
add_subdirectory(${GLUT_ROOT_DIR})

include_directories(${GLUT_INCLUDE_DIRS})
```